config.md

```diff
@@ -148,8 +148,9 @@ Note: Any OPTIONAL field MAY also be set to null, which is equivalent to being a
 
    - **Volumes** *object*, OPTIONAL
 
-     A set of directories which SHOULD be created as data volumes in a container running this image.
-     If a file or folder exists within the image with the same path as a data volume, that file or folder will be replaced by the data volume and never be merged.
+     A set of directories describing where the process is likely write data specific to a container instance.
+     Implementations SHOULD provide mounts for these locations such that application data is not written to the container's root filesystem.
```

Comment:
In this sentence, "implementations" seems to be directed at _runtime_ implementations, right?